src/lib/list/list.spec.ts

```diff
@@ -207,6 +208,62 @@ describe('MatList', () => {
     expect(items.every(item => item._isRippleDisabled())).toBe(true);
   });
 
+  it('should disable item ripples when list ripples are disabled via the input in nav list',
+    fakeAsync(() => {
+      const fixture = TestBed.createComponent(NavListWithOneAnchorItem);
+      fixture.detectChanges();
+
+      const rippleTarget = fixture.nativeElement.querySelector('.mat-list-item-content');
+
+      dispatchMouseEvent(rippleTarget, 'mousedown');
+      dispatchMouseEvent(rippleTarget, 'mouseup');
+
+      expect(rippleTarget.querySelectorAll('.mat-ripple-element').length)
+          .toBe(1, 'Expected ripples to be enabled by default.');
+
+      // Wait for the ripples to go away.
+      tick(1000);
```

Comment:
Why use `1000` here? You should be able to use the default durations similar to the `ripple.spec.ts`

Same for the action list.

Reply:
I didn't want to have one spec file import another spec file so I went with an arbitrarily long timeout that should be enough for the ripples to go away.

Reply:
You don't need to import these from the `ripple.spec.ts` as the default config is just exported from the `RippleRenderer`.